packages/kit/types/index.d.ts

```diff
@@ -87,7 +87,7 @@ export interface Builder {
 	config: ValidatedConfig;
 	/** Information about prerendered pages and assets, if any. */
 	prerendered: Prerendered;
-	/** An array of dynamic (not prerendered) routes */
+	/** An array of all routes (including prerendered) */
```

Comment:
drive-by comment fix - I don't think we need a changeset for this, it's okay to just release it as part of the next Kit release without special mention